### DISCUSSION TITLE
Run the test.py script directly on new images

### DISCUSSION THREAD

**Discussion by adwaykanhere** [junyanz/pytorch-CycleGAN-and-pix2pix#1525] [Q&A] [answered]
Hi,

I have trained a GAN based on your pix2pix Colab notebook. I would like to test the GAN on a set of new images that the model has never seen - This means I haven't run the "combine dataset" function on my new data and I don't have their pairs. 

Will I be able to run the GAN only on these images alone? 

Thanks,

**Comment by junyanz** [comment, marked as answer]
It's possible. You can use `--model tet`. 

![Screen Shot 2023-03-14 at 5 10 56 PM](https://user-images.githubusercontent.com/1924757/225137134-2e22287d-45ca-4be7-83a6-a9f73e37f13d.png)

See [here](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix#apply-a-pre-trained-model-pix2pix) for more details.
